src/App.js

```diff
@@ -1,13 +1,49 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
 import "./App.css";
+import Card from "./../src/components/Card.js";
+import Header from "./../src/components/Header.js";
 
 function App() {
+  const [nasaObj, setNasaObj] = useState({});
+
+  // axios directly in function would rerun the GET request each time the function is run
+  // useEffect - whenever data changes from last render of function, useEffect runs callback. runs one time
+
+
+  useEffect(() => {
+    const dateArray = ["2019-01-01", "2019-05-12", "2019-07-18"]
+
+    dateArray.forEach(newDate => {
+      axios
+      .get(
+        `https://api.nasa.gov/planetary/apod?api_key=21qDXlW15GNFOXLKsjJBrPnCHKPDftSc4Y4S6ce6&date=${newDate}`
+        // "https://api.nasa.gov/planetary/apod?api_key=21qDXlW15GNFOXLKsjJBrPnCHKPDftSc4Y4S6ce6"
+      )
+      .then(res => {
+        const nasaData = res.data;
+        console.log(nasaData);
+        setNasaObj(res.data);
```

Comment:
This is a great way to make multiple requests, however the will all overwrite each other. A solution might be to make our state varaible an array, like `nasaObjs`. Then, inside of here, we can do `setNadaObjs(objs => objs.concat(res.data))`. This way we will add them to the end of the array as they come in. You can then map over them when you make your components.